pkg/minikube/cluster/cluster.go

```diff
@@ -413,19 +414,25 @@ func GetHostDockerEnv(api libmachine.API) (map[string]string, error) {
 
 // GetHostLogs gets the openshift logs of the host VM.
 // If follow is specified, it will tail the logs
-func GetHostLogs(api libmachine.API, follow bool) (string, error) {
+func GetHostLogs(api libmachine.API, follow bool, tail int64) (string, error) {
 	host, err := CheckIfApiExistsAndLoad(api)
 	if err != nil {
 		return "", err
 	}
 
-	if follow {
+	if follow || tail != -1 {
 		c, err := host.CreateSSHClient()
 		if err != nil {
 			return "", err
 		}
 
-		err = c.Shell(logsCmdFollow)
+		if follow && tail == -1 {
```

Comment:
@budhram I am confused about this condition. This is for `$ minishift logs -t -1 -f` right?

Reply:
Only follow specificed.